lfpykernels/kernel_approximator.py

```diff
@@ -590,7 +617,8 @@ def get_kernel(self, probes, Vrest=-65, dt=2**-4,
                 data[h, ] = np.convolve(d, h_delta, 'same')
 
             # subtract kernel offsets at tau == 0:
-            data = data - data[:, cell.tvec == t_X]
+            t_idx_ = np.argmin(np.abs(cell.tvec - t_X))
```

Comment:
Not sure if I understand. Are there cases where `t_X` is not *on* the simulation grid? Maybe we should not allow that. Could add an `assert t_X % dt == 0, mssg`?

Reply:
I'm not sure if it makes sense, but I was just thinking it safer since "==" can sometimes give unexpected results because of round-off errors, like 0.1 + 0.2 == 0.3 is False.